TSC-Charter.md

```diff
@@ -142,18 +114,19 @@ method should be used, such as:
 Multiple-candidate methods may be reduced to simple election by plurality
 when there are only two candidates for one position to be filled. No
 election is required if there is only one candidate and no objections to
-the candidates election. Elections shall be done within the Projects by
-the Collaborators active in the Project.
+the candidate's election. Elections shall be done within the projects by
+the Collaborators active in the project.
 
 The TSC will elect from amongst voting TSC members a TSC Chairperson to
-work on building an agenda for TSC meetings and a TSC Director to represent
-the TSC to the Board for a term of one year according to the Node.js Foundation’s
-By-laws. The Chair and Director may be (but are not required to be)
-the same person. The TSC shall hold annual elections to select a TSC
-Chairperson and Director; there are no limits on the number of terms a
-TSC Chairperson or Director may serve.
+work on building an agenda for TSC meetings and a OpenJS
```

Comment:
As an Impact project, Node.js has two voting members in CPC https://github.com/openjs-foundation/cross-project-council#impact-project-representatives

Should this be changed to two voting members?

Reply:
@trivikr no. The TSC only elects one of the two members. The second member is elected by the CommComm

Reply:
:+1: